core/uploader.go

```diff
@@ -29,15 +29,15 @@ func (bc *ByteCounter) Write(p []byte) (n int, err error) {
 
 func newExponentialBackOffExecutor() *backoff.ExponentialBackOff {
 	backOff := backoff.NewExponentialBackOff()
-	backOff.InitialInterval = 10 * time.Second
-	backOff.MaxInterval = 1 * time.Minute
+	backOff.InitialInterval = 30 * time.Second
+	backOff.MaxInterval = 2 * time.Minute
```

Comment:
I think we can do even more than this - we can more about it succeeding eventually than being quick

Reply:
There's a balance we have to strike here. 

If we try for a very long time indefinitely, we eventually run the risk of running out of threads again depending on length of outage (what we experienced last night). This is a limitation of the current architecture.

If we try for a longer time than what we do today, but give up sooner, then we don't run the risk of running out of threads but we'll lose recorded segments. 

IMO, it's better to do the latter approach and have a path to upload to secondary backup storage (something we need anyways).